File: pset6/mario.py
import cs50

def main():
    size = get_pyramid_size()
    
    for i in range(size):
        for j in range(size-i-1):
            print(" ", end="")
        for j in range(i+1):
            print("#", end="")
        print("  ", end="")
        for j in range(i+1):
            print("#", end="")
        print()

def get_pyramid_size():
    while True:
        print("Enter a pyramid size: ")
        size = cs50.get_int()
        if size > 0 or size < 23:
            break
    return size

if __name__ == "__main__":
    main()
    

File: pset1/mario.c
#include <stdio.h>
#include <stdlib.h>
#include <cs50.h>

int main()
{
   int length;

    do
    {
       printf("Enter amount of rows in the pyramid < 24: ");
       length = GetInt();
    }
    while(length < 0 || length > 23);

    //Creats raws
    for(int i = 0; i < length; i++)
    {
        //Creates spaces
        for(int j = 0; j < length-i-1; j++) 
        {
            printf(" ");
        }
        
        //Creates left side of the pyramid
        for(int j = 0; j <= i; j++) 
        {
            printf("#");
        }
        //Creates center of the pyramid
        printf("  ");

        //Creates right side of the pyramid
        for (int j = 0; j <= i;  j++)
    	{
    		printf("#");
    	}

    	printf("\n");
    }

    return 0;
}

